api/src/main/java/org/openmrs/module/sync2/api/validator/impl/SyncConfigurationInstanceIdValidator.java

```diff
@@ -10,11 +11,20 @@
 public class SyncConfigurationInstanceIdValidator implements SyncConfigurationValidator {
 
     private static final String ERROR_CODE = "sync2.error.instanceIdIsNotSet";
+    private static final String PARENT_URI_ERROR = "sync2.configuration.parentUrl.empty";
 
     @Override
     public void validate(SyncConfiguration syncConfiguration, Errors errors) {
         if (StringUtils.isBlank(syncConfiguration.getGeneral().getLocalInstanceId())) {
             errors.addErrorCode(ERROR_CODE);
         }
+        if (parentInstanceUriIsEmpty()) {
+            errors.addErrorCode(PARENT_URI_ERROR);
+        }
+    }
+
+    private boolean parentInstanceUriIsEmpty() {
```

Comment:
I see this method is used few times. Please move it to SyncUtils class

Reply:
Good idea, thanks.